pages/_index.md

```diff
@@ -0,0 +1,8 @@
+<!-- Template for the top page of a specification directory containing releases -->
+---
+title: "Jakarta Server Pages"
+summary: Jakarta Server Pages defines a template engine for web applications that supports mixing of textual content
+(including HTML and XML) with custom tags, expression language, and embedded Java code, that gets compiled
+into a Jakarta Servlet.
```

Comment:
I think the summary needs to be enclosed in double quotes.

Reply:
And I'm not sure if it can be on multiple lines.

@waynebeaton can you confirm?  (I think this is a hugo question.)

Reply:
The parsed / presented version looks off.

See: https://github.com/lukasj/specifications/blob/jsp/pages/_index.md

I would have expected it to be presented in a table like ...

https://github.com/joakime/specifications/blob/websocket-specs/websocket/_index.md

Reply:
This is really just a file with metadata that's processed by hugo to generate the actual web site data.  I don't know enough about hugo and how this is processed to know what it's going to look like.

Reply:
> And I'm not sure if it can be on multiple lines.

No quotes are required; it can be on multiple lines.

> This is really just a file with metadata that's processed by hugo to generate the actual web site data. I don't know enough about hugo and how this is processed to know what it's going to look like.

Right. It's metadata. The site knows how to take this content and render it. What you're seeing is the GitHub rendering of the Markdown content (Markdown is, AFAICT, a bit of a wild west). I'm pretty sure that if you pull the HTML comment out of your file, GitHub will render it in a similar manner (FWIW, Hugo seems to just ignore the comment)

Reply:
template file has quotes, so I put them there.